templates/superset_config.py

```diff
@@ -101,6 +101,11 @@ class CeleryConfig(object):
 TIME_ROTATE_LOG_LEVEL = "DEBUG"
 ENABLE_TIME_ROTATE = True
 
+# html santization
+HTML_SANITIZATION = os.getenv("HTML_SANITIZATION")
+if not HTML_SANITIZATION and os.getenv("HTML_SANITIZATION_SCHEMA_EXTENSIONS"):
```

Comment:
So HTML sanitization schema extensions can only be added if `html_sanitization` is disabled?

Reply:
Ah it was actually meant to be only if it is enabled. Thank you, I have fixed this.